src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DATAACCESS/MRMFeatureAccessOpenMS.h

```diff
@@ -75,6 +75,8 @@ namespace OpenMS
 
     double getRT() const override;
 
+    double getMetaValue(std::string name) const;
```

Comment:
consider using RangeIM

Reply:
I was looking into applying this, but I think it would require more changes to other parts of OpenSwath. Currently OpenSwath (and some of the other FeatureFinders) stores the leftWidth and rightWidth in the metaValue, which is then used for retrieval at some point. I think the `RangeRT` would be a good replacement, but I think those changes would maybe be for another PR?